test/unit/vl-form-validation-test.js

```diff
@@ -1,19 +1,17 @@
 import {vlElement, define} from '/node_modules/vl-ui-core/dist/vl-core.js';
-import '/node_modules/@govflanders/vl-ui-util/dist/js/util.js';
-import '/node_modules/@govflanders/vl-ui-core/dist/js/core.js';
-import '/lib/form-validation.js';
+import {vlFormValidation} from '/src/vl-form-validation.js';
+
+vlFormValidation.awaitUntilReady().then(() => define('vl-form-validation-test', VlFormValidationTest));
 
 class VlFormValidationTest extends vlElement(HTMLElement) {
   constructor() {
     super(`
       <form data-vl-validate-form>
-          <label class="vl-form__label" for="validation-naam">Naam</label>
-          <div class="vl-input-group">
-              <input name="validation-naam" type="text" data-required="true" data-vl-success-class="success-style"
-                  data-vl-error-class="error-style" data-vl-error-message="Veld &quot;Naam&quot; is verplicht"
-                  data-vl-error-placeholder="validation-naam-error" aria-required="true" aria-invalid="true" />
-          </div>
-          <div class="error-message" data-vl-error-id="validation-naam-error"></div>
+        <label class="vl-form__label" for="validation-naam">Naam</label>
+        <div class="vl-input-group">
+          <input name="validation-naam" type="text" data-required="true" data-vl-success-class="success-style" data-vl-error-class="error-style" data-vl-error-message="Veld &quot;Naam&quot; is verplicht" data-vl-error-placeholder="validation-naam-error" aria-required="true" aria-invalid="true" />
```

Comment:
zijn er testen of die custom class success-style en error-style gezet worden? of is dit te vergaand?

Reply:
We voorzien normaal gezien alleen maar unit testen voor code die we zelf geschreven hebben. De werking hiervan wordt wel getest in de e2e testen.